listenbrainz/webserver/static/js/src/listens/ListenCard.tsx

```diff
@@ -164,6 +164,18 @@ export default class ListenCard extends React.Component<
     }
   };
 
+  isListenReviewable = () => {
+    const { listen } = this.props;
+    const mbids = listen.track_metadata.additional_info;
```

Comment:
We should access nested properties in a safer manner as that path might not exist:
`const additionalInfo = listen.track_metadata?.additional_info` (with the `?.` syntax)